javascript/react/src/components/StepHistoryList.types.ts

```diff
@@ -16,6 +16,7 @@ export interface StepHistoryListProps
 	order?: DisplayOrder
 	onDelete?: (index: number) => void
 	onDuplicateClicked?: (step: Step) => void
+	onSelectOriginalTable?: () => void
```

Comment:
Why does this need a different callback? I'm guessing because the workflow doesn't know if it's in the inputs list or outputs list, but it definitely feels weird to have to differentiate in this component's API.

Reply:
yep you're right. On the outside we're just getting the latest input anyway, so I've moved it inside